### PR TITLE
sync: git: also print merge_cmd

### PR DESCRIPTION
**Pull request by Flowdalic**
It felt strange that the sync operation would print the "git fetch" command, but not the command used to merge the remote's changes into the local working tree.

### CODE REVIEW

File: lib/portage/sync/modules/git/git.py [hunk 1]
```diff
@@ -370,6 +370,8 @@ def update(self):
         if quiet:
             merge_cmd.append("--quiet")
 
+        writemsg_level(' '.join(merge_cmd) + "\n")
```
Comment (thesamesam) [inline lib/portage/sync/modules/git/git.py:373]: Maybe only if not quiet? What do you think?

Comment (Flowdalic) [reply lib/portage/sync/modules/git/git.py:373]: Sounds sensible. Should do the same for the "git fetch" message above?

Comment (thesamesam) [reply lib/portage/sync/modules/git/git.py:373]: I think so yeah

**Comment by Flowdalic** [comment]
New version pushed.

**Comment by thesamesam** [comment]
Thanks!

**Pull request closed without merging**